cmd/utils/flags.go

```diff
@@ -808,6 +808,60 @@ var (
 		Value: "~/klaytn/bin/kcnd",
 	}
 
+	// db migration vars
+	DstDbTypeFlag = cli.StringFlag{
+		Name:  "dst.dbtype",
+		Usage: `Blockchain storage database type ("LevelDB", "BadgerDB", "DynamoDBS3")`,
+		Value: "LevelDB",
+	}
+	DstDataDirFlag = DirectoryFlag{
+		Name:  "dst.datadir",
+		Usage: "Data directory for the databases and keystore",
+		Value: DirectoryString{node.DefaultDataDir()},
```

Comment:
How about removing this default value? Since this dir path is used for the default value of src DB also, this path could be set for both of src DB and dst DB.

Reply:
reveiw reflected